cmake/external/mkldnn.cmake

```diff
@@ -34,7 +34,7 @@ else()
 endif()
 
 if (onnxruntime_USE_MKLML)
-  set(MKLDNN_VERSION_SHORT v0.20)
+  set(MKLDNN_VERSION_SHORT v1.0)
   set(MKLML_URL https://github.com/intel/mkl-dnn/releases/download/${MKLDNN_VERSION_SHORT}/mklml_${MKLML_OS_VERSION_STR}_${MKLML_VERSION}.${MKLML_FILE_EXTENSION})
```

Comment:
The mklml binary isn't available with v1.0 so we need to use the previous version for the mklml url

Reply:
my bad. I changed MKLDNN_VERSION_SHORT to v0.20 so that we  we can download mklml. double checked with --use_mklml